src/utils/getActionCreators.ts

```diff
@@ -0,0 +1,29 @@
+import { Models } from '../../typings/rematch'
+import dispatchPlugin from '../plugins/dispatch'
+import isListener from './isListener';
+
+const { expose: { dispatch } } = dispatchPlugin
+
+export default (models: Models): { [key: string]: Function } =>
+    Object.keys(models).reduce((actionCreators, modelName) => {
+        const { reducers = {} } = models[modelName];
+
+        Object.keys(reducers)
+            .filter(reducerName => !isListener(reducerName))
+            .forEach(reducerName => {
+                const reducer = reducers[reducerName];
+                const type = `${modelName}/${reducerName}`;
+
+                // We have to dynamically create the function like this,
+                // so that the argument name is not minified.
```

Comment:
Interesting. I hadn't thought about the effect of minification here.

Reply:
Yeah it's a bit of a pain. I wrote the initial version of this without the dynamic function creation and ended up with `model/action(e)` in the dropdown of the devtool :confused:

Reply:
Ouch. Learn something every day.